EIPS/eip-FOCIL.md

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
```suggestion
---
eip: 7805
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.